sale_commission/models/settlement.py

```diff
@@ -106,7 +107,7 @@ def _prepare_invoice(self, journal, product, date=False):
         for settlement in self:
             with move_form.invoice_line_ids.new() as line_form:
                 line_form.product_id = product
-                line_form.quantity = 1
+                line_form.quantity = -1 if settlement.total < 0 else 1
```

Comment:
But we don't want negative invoices...

Reply:
I know, at the end it is transformed to a refund. Similary to sale

Reply:
OK, seen.